code/src/components/GameScreen.js

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import { useSelector, useDispatch } from "react-redux";
+import game, { playGame } from "reducers/game";
+import styled from "styled-components";
+import { TextContainer } from "./StartScreen";
+
+export const GameContainer = styled.div`
```

Comment:
Ambitious of you to used styled components in this project. It's great to keep practice on it and it seems like you really got the hang of it. Good job!